jib-core/src/main/java/com/google/cloud/tools/jib/api/TarImage.java

```diff
@@ -35,7 +32,7 @@
  * }</pre>
  */
 // TODO: Add tests once JibContainerBuilder#containerize() is added.
```

Comment:
I feel like we can remove some of these TODOs

Reply:
Agreed, we now have many tests for `containerize()`.